posthog/middleware.py

```diff
@@ -608,6 +608,8 @@ def process_response(self, request, response):
             # clears the cookies that were previously set, except for ph_current_instance as that is used for the website login button
             response.delete_cookie("ph_current_project_token", domain=default_cookie_options["domain"])
             response.delete_cookie("ph_current_project_name", domain=default_cookie_options["domain"])
+            return response
```

Comment:
It's interesting that this is suddenly necessary... and it seems more correct?
@benjackwhite maybe you can have a look?
also see the needed proper logout in `posthog/middleware.py`

Reply:
Ping 😊